### ISSUE TITLE
broken for flutter 2.8

### ISSUE THREAD

**Issue by felix-ht** [yako-dev/flutter-settings-ui#102]
If flutter 2.8 we get the error: 
```
: Error: A value of type 'ListTileThemeData' can't be assigned to a variable of type 'ListTileTheme'.
../…/src/cupertino_settings_item.dart:76
- 'ListTileThemeData' is from 'package:flutter/src/material/list_tile.dart' ('../../flutter/packages/flutter/lib/src/material/list_tile.dart').
package:flutter/…/material/list_tile.dart:1
- 'ListTileTheme' is from 'package:flutter/src/material/list_tile.dart' ('../../flutter/packages/flutter/lib/src/material/list_tile.dart').
package:flutter/…/material/list_tile.dart:1
    final ListTileTheme tileTheme = ListTileTheme.of(context);
```

Fixed with
https://github.com/yako-dev/flutter-settings-ui/blob/dd14938d5c58863b0a0f32897267d35ce6ea8ba4/lib/src/cupertino_settings_item.dart#L80
changed to (and later references changed ListTileThemeData)
`final tileTheme = ListTileTheme.of(context);`

in general i would recommend to avoid typing these types kind of assignments explicitly

**Comment by HobbyDroider** [comment]
The mentioned later reference is in the signature of the method `_iconColor(ThemeData theme, ListTileTheme tileTheme)` change that to `_iconColor(ThemeData theme, ListTileTheme tileTheme)` as a quick fix.

**Comment by superkeka** [comment]
Same issue

**Comment by leszekru** [comment]
Yes, it's broken and 'easy' to quick-fix...

**Comment by azazadev** [comment]
same issue

**Comment by iakrevetkho** [comment]
same issue

**Comment by cirediew** [comment]
Duplicate of #99 Pull request is waiting to be merged #101

**Comment by elirehema** [comment]
Just check this comment here https://github.com/yako-dev/flutter-settings-ui/pull/101#issuecomment-993603256 it can help

**Comment by mrinvestment** [comment]
same issue ... :-(

**Comment by ryandailey100** [comment]
@yadaniyil Please merge fix & release update

**Comment by cirediew** [comment]
@ryandailey100 as stated before, read this https://github.com/yako-dev/flutter-settings-ui/pull/101#issuecomment-993603256

**Comment by yadaniyil** [comment]
Hey @felix-ht @HobbyDroider @superkeka @leszekru @azazadev @cirediew @elirehema @mrinvestment @ryandailey100 . There is a new 2.0.0 version of the package, could you please if your issue is fixed there? Thanks :)

**Comment by elirehema** [comment]
Thanks, @yadaniyil, unfortunately, I have followed the instruction in[ #101 comment](https://github.com/yako-dev/flutter-settings-ui/pull/101#issuecomment-993603256)

**Comment by cirediew** [comment]
@yadaniyil This issue seems to be fixed for me on 2.0.1. Thank you
